tasks/options/jscs.js

```diff
@@ -1,3 +1,6 @@
 module.exports = {
-	all: [ "*.js", "performance/**/*.js" ]
-};
\ No newline at end of file
+	all: [ "*.js", "performance/**/*.js", "scss/variables/**/*.js" ],
+	options: {
+		fix: true
+	}
```

Comment:
I ran it once with fix true no issues. We could get rid of the option now or do we want to keep it ? @arschmitz ?